inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -927,4 +918,23 @@ public function notice_write_permissions() {
 			]
 		);
 	}
+
+	/**
+	 * Validate the items in array to be strings only and preg_quote them.
+	 *
+	 * @param array $items Array to be validated and quoted.
+	 *
+	 * @return array|string[]
+	 */
+	private function validate_array_and_quote( array $items ) {
+		$items_array = array_filter( $items, 'is_string' );
+
+		return array_map(
+			function ( $item ) {
```

Comment:
You can consider using a static anonymous function as you don't need the context ^^'
```suggestion
			static function ( $item ) {
```

Reply:
Good point, Yes this is [micro-optimization](https://www.designcise.com/web/tutorial/what-are-static-anonymous-functions-in-php#when-to-use-static-closures) so why not.
Thanks.